public/pages/SnapshotPolicyDetails/containers/SnapshotPolicyDetails/SnapshotPolicyDetails.tsx

```diff
@@ -259,18 +282,25 @@ export default class SnapshotPolicyDetails extends Component<SnapshotPolicyDetai
       [condition: string]: boolean;
     }
     const notiConditions: NotiConditions = _.get(policy, "notification.conditions");
-    // _.get(policy, "notification.conditions")
     let notiActivities = "None";
     if (notiConditions) {
       notiActivities = Object.keys(notiConditions)
         .filter((key) => notiConditions[key])
         .join(", ");
     }
-    console.log(`sm dev notification ${notiActivities}`);
+
+    let channelDetail = <p>None</p>;
+    if (!!channel?.config.name) {
+      channelDetail = (
+        <EuiLink href={`notifications-dashboards#/channel-details/${channel.config_id}`}>
```

Comment:
This would open in the same window as opposed to a new tab, right? Is that intended? Adding `target="_blank"` here would open a new tab.